pytorch_toolkit/tests/run_model_templates_tests2.py

```diff
@@ -0,0 +1,431 @@
+# Copyright (C) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import argparse
+import fnmatch
+import glob
+import logging
+import os
+import shlex
+import sys
+import tempfile
+import unittest
+import yaml
+
+from collections import Counter
+from copy import copy
+from subprocess import run
+from texttable import Texttable
+
+
+# this is required for import ote in tests during tests discover
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'ote'))
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
+
+KNOWN_DOMAIN_FOLDERS = [
+        'object_detection',
+        'action_recognition_2',
+        'instance_segmentation',
+        'text_spotting',
+        'image_classification',
+        'ote',
```

Comment:
ote is not domain

Reply:
Ilya, `ote` has internal tests. 
At the moment it has tests on service function `merge_dicts_and_lists_b_into_a` -- see the file [pytorch_toolkit/ote/tests/internal_tests_compression_config_merger.py](https://github.com/openvinotoolkit/training_extensions/blob/lb/add_run_model_templates_tests2.py/pytorch_toolkit/ote/tests/internal_tests_compression_config_merger.py)
But Alexander Kozlov asked to add more tests on some inner classes, functions, etc

So, from the "tests point of view" ote _is_ a domain
Please, note that now the folder `ote` has subfolder `tests` that contains script `init_venv.sh` to create special virtual environment for ote tests.

P.S. But I can rename the variable `KNOWN_DOMAIN_FOLDERS -> KNOWN_FOLDERS_WITH_TESTS`

Reply:
I would prefer 'DOMAINS'. 'ote' is not a domain (algo domain). If you need to test it, add corresponding runner.